.github/workflows/tests-fast.yml

```diff
@@ -0,0 +1,110 @@
+name: Tests (Fast)
```

Comment:
<img width="226" alt="image" src="https://github.com/user-attachments/assets/33b00429-e4c1-43bc-a5d4-26f5a20198b4" />

This creates a naming stutter. If we are cleaning this up here, I suggest renaming `Fast:` below to `Tests:` or something?